### DISCUSSION TITLE
Publications: displaying the journal badge over thumbnail picture?

### DISCUSSION THREAD

**Discussion by matonoli** [alshedivat/al-folio#1900] [Q&A]
Currently, when a preview thumbnail is provided using `preview={something.gif}`, it is displayed instead of the journal badge specified in `abbr={Journal}`.
Is there a way to make the badge show ON TOP of the preview thumbnail? As in, overlaid?
If not, how would one go about implementing such a change?
Thanks!

**Comment by alshedivat** [comment]
> Is there a way to make the badge show ON TOP of the preview thumbnail? As in, overlaid?

could you share an image that shows what you mean exactly? it's not clear what "overlaid" version could look like.

**Reply by matonoli** [reply]
Thanks for the quick reply. Absolutely. Very crudely, something like this:
![publication](https://github.com/alshedivat/al-folio/assets/20525594/d978aae6-1509-4e00-9633-80d319682e02)

I don't know if there's an easy way to "stack" objects on top of each other like this, it's been a while since I've done anything with HTML or CSS. Thanks again!
